python/vpc-ec2-local-zones/vpc_ec2_local_zones/vpc_ec2_local_zones_stack.py

```diff
@@ -0,0 +1,182 @@
+import re
+
+from aws_cdk import (
+    Stack,
+    CfnOutput,
+    aws_ec2 as ec2,
+    aws_elasticloadbalancingv2 as elbv2,
+    aws_autoscaling as autoscaling,
+)
+from constructs import Construct
+
+# Local Zone to be used
+LZ_NAME = "us-east-1-atl-1a"
+# VPC CIDR that will be used to create a new VPC
+VPC_CIDR = "172.31.100.0/24"
+# Subnet size of the subnets in the Local Zone
+SUBNET_SIZE = 26
+
+
+class VpcEc2LocalZonesStack(Stack):
+
+    # overwrite availability_zones method used by CDK to determine the
+    # Avaialability Zones or the Local Zone to be used by ec2.VPC construct
+    @property
+    def availability_zones(self):
+       return [LZ_NAME]
+
+    def create_VPC(self):
+        vpc = ec2.Vpc(
+            self,
+            "Vpc",
+            cidr = VPC_CIDR,
```

Comment:
```suggestion
            ip_addresses = ec2.IpAddresses.cidr(VPC_CIDR),
```

As cidr has been deprecated, consider using [ipAddress](https://docs.aws.amazon.com/cdk/api/v2/python/aws_cdk.aws_ec2/Vpc.html#vpc) instead.